packages/firebase_remote_config/lib/src/remote_config.dart

```diff
@@ -0,0 +1,210 @@
+    remoteConfig._lastFetchStatus =
+        LastFetchStatus.values[properties[lastFetchStatusKey]];
+    final RemoteConfigSettings remoteConfigSettings =
+        new RemoteConfigSettings();
+    remoteConfigSettings.debugMode = properties['IN_DEBUG_MODE'];
+    remoteConfig._remoteConfigSettings = remoteConfigSettings;
+    remoteConfig._parameters =
+        _parseRemoteConfigParameters(parameters: properties[parametersKey]);
+    return remoteConfig;
+  }
+
+  static Map<String, RemoteConfigValue> _parseRemoteConfigParameters(
+      {Map<dynamic, dynamic> parameters}) {
+    final Map<String, RemoteConfigValue> parsedParameters =
+        <String, RemoteConfigValue>{};
+    parameters.forEach((dynamic key, dynamic value) {
+      final ValueSource valueSource = ValueSource.values[value['source']];
+      final RemoteConfigValue remoteConfigValue =
+          new RemoteConfigValue._(value['value'].cast<int>(), valueSource);
+      parsedParameters[key] = remoteConfigValue;
+    });
+    return parsedParameters;
+  }
+
+  /// Set the configuration settings for this RemoteConfig instance.
+  ///
+  /// This can be used for enabling developer mode.
+  Future<void> setConfigSettings(
+      RemoteConfigSettings remoteConfigSettings) async {
+    await _channel
+        .invokeMethod('RemoteConfig#setConfigSettings', <String, dynamic>{
+      'debugMode': remoteConfigSettings.debugMode,
+    });
+    _remoteConfigSettings = remoteConfigSettings;
+    return new Future<void>.value();
+  }
+
+  /// Fetches parameter values for your app. Parameter values may be from
+  /// Default Config (local cache) or Remote Config if enough time has elapsed
+  /// since parameter values were last fetched from the server. The default
+  /// expiration time is 12 hours. Expiration must be defined in seconds.
+  Future<void> fetch({Duration expiration: const Duration(hours: 12)}) async {
+    try {
+      final Map<dynamic, dynamic> properties = await _channel.invokeMethod(
+          'RemoteConfig#fetch',
+          <dynamic, dynamic>{'expiration': expiration.inSeconds});
+      _lastFetchTime =
+          new DateTime.fromMillisecondsSinceEpoch(properties[lastFetchTimeKey]);
+      _lastFetchStatus = LastFetchStatus.values[properties[lastFetchStatusKey]];
+    } on PlatformException catch (e) {
+      _lastFetchTime =
+          new DateTime.fromMillisecondsSinceEpoch(e.details[lastFetchTimeKey]);
+      _lastFetchStatus = LastFetchStatus.values[e.details[lastFetchStatusKey]];
+      if (e.code == RemoteConfig.fetchFailedThrottledKey) {
+        print('fetch failed throttled');
+        final int fetchThrottleEnd = e.details['FETCH_THROTTLED_END'];
+        throw new FetchThrottledException._(endTimeInMills: fetchThrottleEnd);
+      } else {
+        print('fetch failed unknown');
+        throw new Exception('Unable to fetch remote config');
+      }
+    }
+    print('fetch succeeded');
+    return new Future<void>.value();
```

Comment:
You should be able to just leave this off I think

Reply:
Done